infra/modules/azure_cosmos_account/package.json

```
@@ -3,5 +3,8 @@
   "version": "0.3.1",
   "private": true,
   "provider": "azurerm",
-  "description": "Provisions an Azure Cosmos DB account with monitoring and network settings"
+  "description": "Provisions an Azure Cosmos DB account with monitoring and network settings",
+  "scripts": {
+    "test-integration": "terraform test -filter='tests/integration.tftest.hcl'"
```

what about

```suggestion
    "test:infra": "terraform test -filter='tests/integration.tftest.hcl'"
```

since we're filtering only *.tf files we may choose a more specific naming (`test-integration` may cover typescript packages as well). Apart if we want to run _all_ integration tests despite their origin.